packages/velocity/src/helpers/velocity-helpers.ts

```diff
@@ -172,7 +182,7 @@ export function getTitle(
   authentication: UserSession,
   label: string,
   url: string
-): Promise<string> {
+): Promise<any> {
```

Comment:
Could the return be typed as something like
```ts
interface IVelocityTitle {
  label: string;
  titles: string[];
}
```
so that we get the benefit of TypeScript's type checking?